apps/design/backend/src/globals.ts

```diff
@@ -22,6 +23,41 @@ export const NODE_ENV = unsafeParse(
   process.env.NODE_ENV ?? 'development'
 );
 
+/* istanbul ignore next - @preserve */
+function requiredProdEnvVar<Fallback>(
+  name: string,
+  devFallback: Fallback
+): string | Fallback {
+  const envVar = process.env[name];
+  if (envVar) {
+    return envVar;
+  }
+
+  assert(NODE_ENV !== 'production', `Env var ${name} required in production.`);
+
+  return devFallback;
+}
+
+/* istanbul ignore next - @preserve */
+export function baseUrl(): string {
+  return requiredProdEnvVar('BASE_URL', `http://localhost:${PORT}`);
+}
+
+/* istanbul ignore next - @preserve */
+export function auth0ClientId(): string {
+  return requiredProdEnvVar('AUTH0_CLIENT_ID', '');
+}
+
+/* istanbul ignore next - @preserve */
+export function auth0IssuerBaseUrl(): string {
+  return requiredProdEnvVar('AUTH0_ISSUER_BASE_URL', '');
+}
+
+/* istanbul ignore next - @preserve */
+export function auth0Secret(): string {
+  return requiredProdEnvVar('AUTH0_SECRET', '');
+}
```

Comment:
Once this is merged, will there be a way to log in in dev?

Reply:
Not yet - dev will current bypasses auth a, but hoping to get it running against a dev Auth0 instance soon 🤞🏿